### DISCUSSION TITLE
Can't seem to log into SMTP Server. SSL is required.

### DISCUSSION THREAD

**Discussion by wlbillmartin** [berrystephenw/OctoText#15] [General]
OctoPrint Log attached with error message at bottom.
[octoprint.log-3.txt](https://github.com/berrystephenw/OctoText/files/6238456/octoprint.log-3.txt)

**Comment by wlbillmartin** [comment]
Tested with snapshot disabled (saw this on another discussion item) with same results.

**Comment by natefrog69** [comment]
It seems to me like this plugin uses STARTTLS and not SSL.  Does your mail server use STARTTLS?  It would have a separate port number than SSL.  Outlook.com uses STARTTLS and from the looks of things the plugin author wrote the plugin specifically for outlook.com and would need to add additional encryption methods (SSL) in order to use mail servers that don't use STARTTLS, but I may be wrong.  Gmail uses both with a separate port number for each.  Hopefully the author can let us know.

**Reply by berrystephenw** [reply]
Sort of. The plugin uses SMTP and there is a port that is usually assigned for the purpose of devices that use external services to access the mail server. I used Outlook because they allowed SMTP access (some mail providers don't) and it was free. TTLS is the protocol that is used for authentication.

**Comment by wlbillmartin** [comment]
My only access to MY SMTP server (mail.wlbillmartin.com) requires SSL at port 465. Is this possible? (I know the port is, just the SSL)
 
Thanks!
B
 
Sir William Louis "Bill" Martin Jr., ATP, Capt. (Ret.), OMS
Active Participant - Planet Earth
AKA:
William P. Lewis
Comedian/Radio Personality/Actor
http://www.williamplewis.com/
Phone: 949.933.6230
Fax: 909.510.8253
 
-----Original Message-----

**Reply by berrystephenw** [reply]
Hi Bill, it's possible - just not the way the code is working now!
You could set up a new email account that is only used for OctoText and then you can send the emails to your server. I do this personally and it's a great way to keep track of only the messages from the plugin; my sent mail folder at OctoText@outlook.com has a record of everything without clogging up my primary email.
